omnimatter_compression/prototypes/compress-recipes.lua

```diff
@@ -347,9 +347,9 @@ end
 -- IS VOID? check for the word void in recipe name or products --
 -----------------------------------------------------------------
 local is_void = function(recipe)
-  if string.find(recipe.name, "void") then
+  if string.find(recipe.name, "void") or string.find(recipe.name, "flaring") or string.find(recipe.name, "incineration")then
     return true
-  elseif string.find(recipe.normal.results[1].name, "void") then
+  elseif recipe.normal.results and (string.find(recipe.normal.results[1].name, "void") or string.find(recipe.normal.results[1].name, "flaring") or string.find(recipe.normal.results[1].name, "incineration"))then
```

Comment:
this line represents the results name, so things like "angels-void", "void" ect get captured if the recipe name is not captured, this was due to shenanigans with Kras2. So does flare stack have a consistent "void" result, or is it just the name? It would be cleaner if not both types are used if possible.

Reply:
Yep, that check is not needed here since Flare Stack just uses an empty results table